src/traces/scattermapbox/hover.js

```diff
@@ -19,6 +19,8 @@ module.exports = function hoverPoints(pointData, xval, yval) {
         xa = pointData.xa,
         ya = pointData.ya;
 
+    if(cd[0].placeholder) return;
```

Comment:
yep placeholder is ok. Perhaps we can expand a little in that original placeholder comment. I still didn't quite understand when they would be necessary